darkhorse/generate_dh_keywords.pl

```diff
@@ -0,0 +1,773 @@
+#!/usr/bin/perl
```

Comment:
This is fine for now as DH doesn't have a github repository but note that there is a high change to get out of sync, not sure if this is important, between DH versions. Thus, I will normally suggest going on the github submodule route.